Snake/BusinessLayer/AI.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+
+namespace BusinessLayer
+{
+    public class AI
+    {
+        private Direction currentDirection;
+        private int X;
+        private int Y;
+        Random randomNumber = new Random();
+        public int XCoordinate
+        {
+            get
+            {
+                return X;
+            }
+            set
+            {
+                X = value;
+            }
+        }
+
+        public int YCoordinate
+        {
+            get
+            {
+                return Y;
+            } set
+            {
+                Y = value;
+            }
+        }
+
+        public AI()
+        {
+            X = -1;
+            Y = -1;
+        }
+
+        public bool SpawnAI(int XBorder, int YBorder)
+        {
+            if (XBorder < 0 || YBorder < 0)
+            {
+                return false;
+            }
+            this.X = randomNumber.Next(XBorder - 1);
+            this.Y = randomNumber.Next(YBorder - 1);
+           // currentDirection = Direction.Right;
+
+            return true;
+        }
+
+
+
+        public void MoveAI()
+        {
+
+            bool Opposite = true;
+            int rnd = 0;
+            while (Opposite != false)
+            {
+               // Console.Write("I AM HERE");
+                rnd = randomNumber.Next(0,4);
+                if (rnd == 0 && currentDirection == Direction.Left)

```

Comment:
All these if statements could be a single if statement with || separating each one as they all do the same thing.